CONTRIBUTING.md

```diff
@@ -87,3 +89,343 @@ generated using the pre-commit hooks for
+
+    ```yaml
+    deploy:
+      helm:
+        releases:
+          - name: fiftyone-teams-app
+            setValueTemplates:
+              # imagePullSecrets:
+              #   - name: regcred
+    ```
+
+1. For each service
+
+    * `apiSettings`
+    * `appSettings`
+    * `pluginsSettings`
+    * `teamsAppSettings`
+
+   update the `image.repository` value to be `<REPOSITORY>/<IMAGE_NAME>` and
+   the corresponding `image.tag` value.  For example for the version `1.6.0` at `dev7`
+
+    ```yaml
+    apiSettings:
+      image:
+        repository: us-central1-docker.pkg.dev/computer-vision-team/dev-docker/fiftyone-teams-api
+        tag: v1.6.0.dev7
+    appSettings:
+      image:
+        repository: us-central1-docker.pkg.dev/computer-vision-team/dev-docker/fiftyone-app
+        tag: v1.6.0.dev7
+    pluginsSettings:
+      image:
+        repository: us-central1-docker.pkg.dev/computer-vision-team/dev-docker/fiftyone-app
+        tag: v1.6.0.dev7
+    teamsAppSettings:
+      image:
+        repository: us-central1-docker.pkg.dev/computer-vision-team/dev-docker/fiftyone-teams-app
+        # Note: the naming convention for the image `fiftyone-teams-app` differs from the other images `fiftyone-app`, `fiftyone-app` and `fiftyone-teams-api`
+        # the others are `v1.6.0.dev7` (not `.dev7` vs `-dev7`)
+        tag: v1.6.0-dev.7
+    ```
+
+    > _Note:_ To see the available tags for each image, see
+    > [https://console.cloud.google.com/artifacts/docker/computer-vision-team/us-central1/dev-docker?project=computer-vision-team](https://console.cloud.google.com/artifacts/docker/computer-vision-team/us-central1/dev-docker?project=computer-vision-team)
+
+1. Run skaffold
+
+    ```shell
+    skaffold dev
+
+    # Or with the optional flag
+    # skaffold dev --keep-running-on-failure
+    ```
+
+### Accessing the k8s resources
+
+There are two ways to access resources within the minikube cluster:
+
+* Ingress (recommended)
+* Port Forward
+
+#### Ingress
+
+See
+[Using `minikube tunnel`](https://minikube.sigs.k8s.io/docs/handbook/accessing/#using-minikube-tunnel).
+
+1. Enable the minikube addon `ingress`
+
+    ```shell
+    minikube addons enable ingress
+    ```
+
+1. Install the app via skaffold (see above)
+1. Start the minikube tunnel (and provide sudo password when prompted)
+
+    ```shell
+    $ minikube tunnel
+    ✅  Tunnel successfully started
+
+    📌  NOTE: Please do not close this terminal as this process must stay alive for the tunnel to be accessible ...
+
+    ❗  The service/ingress fiftyone-teams-fiftyone-teams-app requires privileged ports to be exposed: [80 443]
+    🔑  sudo permission will be asked for it.
+    🏃  Starting tunnel for service fiftyone-teams-fiftyone-teams-app.
+    Password:
+    ```
+
+1. Add an entry in to your `/etc/hosts` file to resolve the DNS name
+   `fiftyone.internal` to resolve to localhost
+
+    ```txt
+    127.0.0.1       fiftyone.internal
+    ```
```

Comment:
could we just use `local.fiftyone.ai` and `api.local.fiftyone.ai`, since those DNS addresses already exist?

Reply:
Sure! I wasn't aware that those DNS existed.

Reply:
I'll replace the Auth0 app configuration adding `https` protocol for local.fiftyone.ai and api.local.fiftyone.ai.  Currently, only the `http` protocol is configured.